packages/volto/news/3740.feature

```diff
@@ -0,0 +1 @@
+Add component to show related items, which can be enabled using the `showRelatedItems` setting. @wesleybl
```

Comment:
@sneridagh this could be considered a breaking change. Imagine a client project where the editors "try out" this new feature and the visual design is broken. For instance, we need designs for this for VLT, DLR, FZJ, etc...

Technically it is a new feature but it has the potential to break client expectations. We either have to mark those issues always as breaking or we have to change our internal policy and very closely monitor feature releases internally.

Reply:
@tisto Yeah, I'm confused by your comment. This is why we added the setting and made it disabled by default, so that it is not a breaking change. It's a setting in the Volto config for developers, not for editors.